### ISSUE TITLE
Exception during startup not supported

### ISSUE THREAD

**Issue by rjahn** [sibvisions/reactUI#429]
If you receive a

[
  {
    "name": "message.error",
    "silentAbort": false,
    "title": "Error",
    "message": "Something went wrong!"
  }
]

during startup, you only see an endless loading progress circle.

**Comment by maximilianweidenauer** [comment]
how do I trigger such an error during startup?

**Comment by rjahn** [comment]
Simply change the DemoApplication:

```
public DemoApplication(UILauncher pLauncher) throws Throwable
{
	super(pLauncher);
		
if (true) throw new Exception("Something went wrong!"); 

```

**Comment by maximilianweidenauer** [comment]
![image](https://github.com/sibvisions/reactUI/assets/66732863/82138c80-fc12-41ac-9ef0-984814a00d79)
works for me. possibly also fixed while fixing FS#8706?

**Comment by maximilianweidenauer** [comment]
This ticket opened 15 Nov. the ticket that probably fixed this issue was closed 23 Nov.
